cmd/color.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/fatih/color"
+	"github.com/mattn/go-isatty"
+	"github.com/spf13/cobra"
+)
+
+var noColorFlag bool = true
+
+func addColorFlag(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&noColorFlag, "no-color", true, "Disable colorized output.")
```

Comment:
This seems nice, why disable it by default?

Reply:
Because it's:
1) experimental
2) not fully implemented in terms of covered opportunities to colorize

We can make sure that it works and add more colors, than switch it on be default, and go full peacock.